.github/workflows/qemu.yml

```diff
@@ -0,0 +1,121 @@
+# Builds project on qemu with custom hmat settings
+name: BasicBuilds
+
+on: workflow_call
+
+env:
+  CI_BRANCH : "${{ github.head_ref || github.ref_name }}"
+  CI_REPO : "https://github.com/${{github.repository}}"
+
+permissions:
+  contents: read
+
+jobs:
+  qemu-build:
+    name: Qemu
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+    - name: Enable KVM
+      run: |
+        echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+        sudo udevadm control --reload-rules
+        sudo udevadm trigger --name-match=kvm
+    - name: Install qemu
+      run: |
+        sudo apt update && sudo apt install -y qemu-system genisoimage qemu-utils
+    - name: Run ssh-keygen
+      run: ssh-keygen -b 4096 -N '' -f ~/.ssh/id_rsa
+    - name: Generate iso with user info
```

Comment:
Can we leave it for now? It is a temporary solution anyway (until we have a proper scripts so we can run them on self-hosted runners as well). Also, it should just work if you copy the content to the 'run' and execute it locally (only need to do that once to generate the iso).

Reply:
@KFilipek I think we can leave it as it is (since it is just working), as we will use this only to allow working on different predefined memspaces - we should still work on a final solution and replace this workflow when we are ready

Reply:
In both cases it will work just simply as "copy-paste", but I understand your point of view. From my side, it looks okay.